dockerfiles/Samtools/Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM bashell/alpine-bash:latest
```

Comment:
happy with this- but maybe we could doc somewhere why different distros needed for different tools. eg plain `alpine:latest` above

Reply:
The preference is always alpine as it is only 5.6 mb in size. We are forced to include bash because Cromwell injects its own `entrypoint` of `/bin/bash`. Some tools are using ubuntu simply because their requirements could not be easily met in alpine.  BWA for example would not build for versions older than 0.7.16 because of a missing type `int64_t`.

Happy to document via comments in the docker file

Reply:
+1